channeldb/migration_01_to_11/codec.go

```diff
@@ -172,7 +172,8 @@ func WriteElement(w io.Writer, element interface{}) error {
 		}
 
 	case lnwire.Message:
-		if _, err := lnwire.WriteMessage(w, e, 0); err != nil {
+		_, err := lnwire.WriteMessage(w, e, lnwire.ProtocolVersionLegacy)
```

Comment:
In https://github.com/lightningnetwork/lnd/commit/633cf78a9d2fd7d5034c314d88d2adcaea58fb1a, doesn't this change in `channeldb/migration18/enclosed_codec.go` need to be `ProtocolVersionTLV` to migrate into the new format?

Reply:
You mean some portion of a commit we partially undone? They may have been the case, the end state is correct though. In any case this PR is going away in favor of a mega combined version so it can be tested all at once properly.